### PR TITLE
Adding readme. 

### LINKED ISSUE #1: README.md with relevant setup information
Also include how to get the updated views to show up and seed the database!

### PR DESCRIPTION
**Pull request by jayasi**
This closes #1

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+1. Fork and clone the repository.
+2. Run `bundle install`.
+3. Run the migrations.
+4. Set up your config variables for aws. This can be done as follows: 
+   * <code> export AWS_ACCESS_KEY_ID='your_aws_access_id' </code>
+   * <code> export AWS_SECRET_ACCESS_KEY='your_aws_access_key'</code>
+   * <code> export S3_BUCKET_NAME='your_s3_bucket_name'</code>
+4. Run `rake db:setup`. This would load some test data that we have populated. 
+5. Running `rake db:seed_with_images` would load default images for you IF you have them in your S3 bucket.
```
Comment (kgrz) [inline readme.md:20]: Is this the same S3 bucket getting exported above? What does Paperclip use?

Comment (kgrz) [inline readme.md:20]: s/expired/exported

Comment (jayasi) [inline readme.md:20]: Expired? I'm not sure if I understand this @kgrz . 

We're only using one s3 bucket for paperclip as well as seeding our images. 
Paperclip is also using s3. Though that part is in a separate branch.

Comment (kgrz) [inline readme.md:20]: I meant exported. I was on mobile when I added that comment and I couldn't spot an "edit" button. Apologies.

> We're only using one s3 bucket for paperclip as well as seeding our images. 
> Paperclip is also using s3. Though that part is in a separate branch.

Okay,  understood.

**Pull request merged**